packages/expo-sensors/src/Pedometer.ts

```diff
@@ -47,7 +96,9 @@ export function watchStepCount(callback: PedometerUpdateCallback): EventSubscrip
  * As [Apple documentation states](https://developer.apple.com/documentation/coremotion/cmpedometer/1613946-querypedometerdatafromdate?language=objc):
  * > Only the past seven days worth of data is stored and available for you to retrieve. Specifying
  * > a start date that is more than seven days in the past returns only the available data.
- * @platform ios
+ *
+ * As [Google documentation states](https://developer.android.com/health-and-fitness/guides/recording-api):
+ * > LocalRecordingClient stores up to 10 days of data.
```

Comment:
```suggestion
 * > `LocalRecordingClient` stores up to 10 days of data.
```

Let's apply inline syntax highlight here.